src/components/MainNav.js

```diff
@@ -1,25 +1,35 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { makeStyles } from "@material-ui/core/styles";
 import BottomNavigation from "@material-ui/core/BottomNavigation";
 import BottomNavigationAction from "@material-ui/core/BottomNavigationAction";
 import WhatshotIcon from "@material-ui/icons/Whatshot";
 import MovieIcon from "@material-ui/icons/Movie";
 import SearchIcon from "@material-ui/icons/Search";
 import TvIcon from "@material-ui/icons/Tv";
+import { useHistory } from "react-router-dom";
 
 const useStyles = makeStyles({
   root: {
-    width: 500,
+    width: "100%",
     position: "fixed",
     bottom: 0,
     backgroundColor: "#2d313a",
     zIndex: 100,
+    padding: 5,
   },
 });
 
 export default function SimpleBottomNavigation() {
   const classes = useStyles();
   const [value, setValue] = React.useState(0);
```

Comment:
You should just import `useState` instead of using `React.useState`, just the same as you did with `useEffect`. Maybe you got this code from somewhere else. If so, please don't just copy paste code, try to to think about what it's doing :)

Reply:
I imported useState und useEffect without using React.useState.